src/main/java/com/savvato/tribeapp/services/ConnectServiceImpl.java

```diff
@@ -102,7 +99,7 @@ public void connect(ConnectIncomingMessageDTO incoming) {
 
     public ConnectOutgoingMessageDTO handleConnectionIntent(String connectionIntent, Long requestingUserId, Long toBeConnectedWithUserId) {
         if (connectionIntent == "") {
-            List<Long> recipients = new ArrayList<>(Arrays.asList(toBeConnectedWithUserId));
+            List<Long> recipients = new ArrayList<>(Collections.singletonList(toBeConnectedWithUserId));
```

Comment:
Any specific reason you used Collections.singletonList rather than Array.asList?

Reply:
I think I was experimenting with the Collections.singletonList method and accidentally modified it here. I'll undo this change.